src/textOnlyPackages/src/microsoft.netcore.platforms/1.1.0/package/services/metadata/core-properties/2754f4a272e54a1f95223526b16e017b.psmdcp

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think psmdcp files are usually generated during pack, does this need to be checked in?

Reply:
Removed.